File: 268-missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let totalSum = 0;
    let sum = 0;

    for(let i = 0; i <= nums.length - 1; i++) {
        sum += nums[i];
    }

    for(let i = 1; i <= nums.length; i++) {
        totalSum += i;
    }
    return totalSum - sum;
};

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum = 0;
        int totalSum = 0;
        for(int i = 0; i <= nums.length - 1; i++) {
            sum += nums[i];
        }
        for(int i = 0; i <= nums.length; i++) {
            totalSum += i;
        }
        return totalSum - sum;
    }
}